### ISSUE TITLE
TODO: Perform Stochastic Reconfiguration Optimization

### ISSUE THREAD

**Issue by hz-xiaxz** [hz-xiaxz/FFS-julia#30]
refer to Becca and Sorella 6.4. Is it possible to use fine-tuned optimizer in Julia to schedule the learning rate?

**Comment by hz-xiaxz** [comment]
Workflow: use Carlo.jl to perform Energy and Og results. Use these results to update the parameter g.

**Comment by hz-xiaxz** [comment]
resolved in (#34)
